src/generator/AutoRest.Ruby.Azure/TemplateModels/AzureMethodTemplateModel.cs

```diff
@@ -64,30 +64,39 @@ public bool IsPageable
         }
 
         /// <summary>
-        /// Returns invocation string for next method async.
+        /// Returns string for next method assigned to the Page.
```

Comment:
>string [](start = 20, length = 6)

should we say "Returns the code which assigns the page.next_method" or something like that?